tgui/packages/tgui/styles/components/TextArea.scss

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2020 Aleksej Komarov
+ * SPDX-License-Identifier: MIT
+ */
+
+@use '../base.scss';
+@use '../functions.scss' as *;
+
+$border-color: #88bfff !default;
+$border-radius: base.$border-radius !default;
+
+.TextArea {
+  overflow: hidden;
+  position: relative;
+  display: inline-block;
+  height: "100%";
+  border: 1px solid $border-color;
+  border: 1px solid rgba($border-color, 0.75);
+  border-radius: $border-radius;
+  color: #fff;
+  background-color: #000;
+  background-color: rgba(0, 0, 0, 0.75);
+  margin-right: 2px;
+  line-height: 17px;
+  overflow: visible;
+}
+
+/* Cut and paste to auto resize the textarea */
+.textarea-container > div, .textarea-container > textarea {
+  word-wrap: break-word; /* make sure the div and the textarea wrap words in the same way */
+  box-sizing: border-box;
+  padding: 2px;
+  width: 100%;
+}
+
+.textarea-container > textarea {
+  overflow: hidden;
+  position: absolute;
+  height: 100%;
+}
+
+.textarea-container > div {
+  padding-bottom: 1.5em; /* A bit more than one additional line of text. */
+  visibility: hidden;
+}
```

Comment:
You can merge these blocks with the other ones below, after you fix the selectors.